Core/src/org/sleuthkit/autopsy/datamodel/FileTypesByExtension.java

```diff
@@ -358,7 +362,8 @@ private String createQuery(FileTypesByExtension.SearchFilterInterface filter) {
             // We will end up with a query that looks something like this:
             // OR LOWER(name) ~ '(\.zip|\.rar|\.7zip|\.cab|\.jar|\.cpio|\.ar|\.gz|\.tgz|\.bz2)$')
             query += "OR LOWER(name) ~ '(\\";
-            query += StringUtils.join(filter.getFilter(), "|\\");
+            query += StringUtils.join(filter.getFilter().stream()
+                    .map(String::toLowerCase).collect(Collectors.toList()), "|\\");
```

Comment:
This is pretty nit picky, but I would use Collectors.joining() instead of StringUtils.join, since there is already a stream involved.